### DISCUSSION TITLE
Show form value as part of a static field

### DISCUSSION THREAD

**Discussion by rvanschoren** [vueform/vueform#382] [Questions] [answered]
I want to create a "Are you sure you want to delete <name>?" form where I would like to set <name> to a certain value.
The form format is JavaScript object of JSON (either is ok).

I could supply the value for name in the form data (via load). Is there a way I can reference the value in the static text?

**Comment by adamberecz** [comment]
Yes, you can access it like this with [runtime compiler](https://github.com/vitejs/vite/discussions/4158#discussioncomment-1282397):

```js
const schema = {
  "other_el": {
    "type": "text",
    "default": "Text",
  },
  "static": {
    "type": "static",
    "content": {
      "template": "<b>{{ el$.form$.el$('other_el')?.value }}</b>",
      "props": ['el$']
    }
  }
}
```

If you don't need JSON necessarily, you can use `render()` function without having to include the runtime compiler.

**Reply by rvanschoren** [reply]
Thanks. Do you happen to have an example that uses the render() function?

**Reply by adamberecz** [reply, marked as answer]
Sure, here it is:

```js
import { h } from 'vue';

const schema = {
  other_el: {
    type: 'text',
    default: 'Text',
  },
  static: {
    type: 'static',
    content: {
      props: ['el$'],
      render() {
        return h('b', this.el$.form$.el$('other_el').value);
      },
    },
  },
};
```

**Reply by rvanschoren** [reply]
Thank you so much! I'm so happy I found VueForm.

**Reply by cp-bwg** [reply]
Is there a way to have a numeric or text input bind to the slider value ?  I would like them to be able to type or use the slider, and they bind.  Couldn't find a way to do this in the builder demo.
